docs/customer-360/messaging_modernization/source.sql

```diff
@@ -0,0 +1,11 @@
+CREATE SOURCE CONNECTOR RabbitMQ WITH (
```

Comment:
Seems this file is not used?

Reply:
Correct.  When ksqlDB-connect integration is available, we will switch all recipes to use source.sql instead of source.json